extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -1010,6 +1041,23 @@ public void verifyPlanningErrors()
       assertThat(e, expectedValidationErrorMatcher);
     }
 
+    protected void verifyLookupLoadingInfoInTaskContext(Map<String, Object> context)
+    {
+      String lookupLoadingMode = context.get(PlannerContext.CTX_LOOKUP_LOADING_MODE).toString();
```

Comment:
Can this never be null? Will the context always have the lookup loading mode set?

Reply:
Yes. This is the same reasoning as our discussion on this thread: https://github.com/apache/druid/pull/16358#discussion_r1593339497

Reply:
Fair enough.